src/main.ts

```diff
@@ -1,29 +1,39 @@
-import csvparse from 'csv-parse/lib/sync';
-import * as core from '@actions/core';
-import * as exec from '@actions/exec';
+import csvparse from 'csv-parse/lib/sync'
+import * as core from '@actions/core'
+import * as exec from '@actions/exec'
 
 async function run(): Promise<void> {
   try {
-    const source: string = core.getInput('src');
+    const source: string = core.getInput('src')
+    const dockerConfigPath: string =
+      core.getInput('docker-config-path') || '/home/runner/.docker/config.json'
```

Comment:
The inputs need to be added to the [action.yml](https://github.com/akhilerm/tag-push-action/blob/main/action.yml#L4), else github action will error out with unrecognized input fields.

Reply:
Done, thank you!